### ISSUE TITLE
Cant seem to connect to "LoadLibraryW" 

### ISSUE THREAD

**Issue by UrMomGe** [Escartem/AmongUsCheatClient#8]
Call to LoadLibraryW in remote process failed.

Don't know what to do with this, and could use some help ^^

**Comment by Escartem** [comment]
Did you check #1 to make sure it fixe your issue?

**Comment by HaxsDev** [comment]
I'm actually getting the same error, and I referenced the #1 to make sure it fixed my issue. I tried using the 9.9s and 9.22s and neither worked, and I tried using extreme injector and I get returned NULL (injection failed). I have no idea.

**Comment by Escartem** [comment]
After doing some testing, I noticed that one of my computers get this issue, so I’m guessing that the error might be caused by a computer setting or something similar.

**Comment by kyllian-vssr** [comment]
Same problem with self build computer (AMD R5 2600X / 16 GB / AMD RX 5500 XT / SOME SSD-NVMe, Win10 pro, Antivirus deactivated).

I tried with "Extrem Injector" injecting aucc-0.0.3-beta2.dll, the menu opens, but, if i click in one buton when the game starts, it crash (unexcepted xD) but yes, problem always here.

**Comment by Escartem** [comment]
And you are using version 2020.9.22s of the game ?

**Comment by kyllian-vssr** [comment]
Yes, tried all versions of your client on 2 computers (this one, and my
low-end laptop ! :/

**Comment by Escartem** [comment]
And do you have vc C++ 2015-2019 x86 x64 installed ?

**Comment by kyllian-vssr** [comment]
As far as I can remember yes, at least on my big pc. But I'm not at home,
I'm working :), I'll check this in 8/9 hours :)

**Comment by Escartem** [comment]
Okay tell me if it work !
